src/Kobo/SystemDialog.cpp

```diff
@@ -63,7 +67,10 @@ class SystemWidget final
   void ExportUSBStorage();
   void IncreaseBacklightBrightness();
   void DecreaseBacklightBrightness();
-  void UpdateBacklightButtons(int percent);
+  void UpdateBacklightBrightnessButtons(int percent);
+  void IncreaseBacklightColourTemperature();
+  void DecreaseBacklightColourTemperature();
```

Comment:
What exceptions do these methods throw? If not at all, add `noexcept`!

Reply:
I've added noexcept.